cms/static/js/views/active_video_upload_list.js

```
@@ -162,13 +184,43 @@ define(
                 if (this.onFileUploadDone) {
                     this.onFileUploadDone(this.collection);
                     this.clearSuccessful();
+                    this.fileErrorMsg = null;
```

Why we need this? We should hide error message before a new upload.